test/cli/json_descriptor_test.go

```diff
@@ -19,9 +19,9 @@ func TestJsonDescriptor(t *testing.T) {
 				"GRYPE_CHECK_FOR_APP_UPDATE": "false",
 			},
 			assertions: []traitAssertion{
-				assertInOutput(`"CheckForAppUpdate": false`), // app config
-				assertInOutput(`"db": {`),                    // db status
-				assertInOutput(`"built":`),                   // db status
+				assertInOutput(`"check-for-app-update": false`), // app config
+				assertInOutput(`"db": {`),                       // db status
+				assertInOutput(`"built":`),                      // db status
```

Comment:
question: What do the comments (both `db status`) on these two lines mean?

Reply:
I'll add more verbiage here. Essentially this is saying that these assertions are for the `db` block in the json output.